styleguide-content/komponenter/knapper.md

```diff
@@ -1,10 +1,10 @@
 Knapper gir brukerne mulighet for å gjøre ulike typer handlinger i brukergrensesnittet, som:
 
--   Starte en prosess. F.eks. «Bli kunde», «Sjekk pris», «Meld skade».
--   Gi en kommando til systemet. F.eks. «Skriv ut», «Last ned», «Betal», «Bestill».
--   Beslutte noe. F.eks. «Kjøp», «Aksepter», «Overfør».
--   Navigere i brukergrensesnittet. F.eks. «Neste», «Fortsett», «Søk».
--   Endre modus eller visning. F.eks. «Redigér» for å gå til redigeringsmodus, «Vis flere» for å ekspandere en liste.
+*   Starte en prosess. F.eks. «Bli kunde», «Sjekk pris», «Meld skade».
+*   Gi en kommando til systemet. F.eks. «Skriv ut», «Last ned», «Betal», «Bestill».
+*   Beslutte noe. F.eks. «Kjøp», «Aksepter», «Overfør».
+*   Navigere i brukergrensesnittet. F.eks. «Neste», «Fortsett», «Søk».
+*   Endre modus eller visning. F.eks. «Redigér» for å gå til redigeringsmodus, «Vis flere» for å ekspandere en liste.
```

Comment:
I didn't do this...

Reply:
prettier?

Reply:
Jeg tipper det... 🤷‍♂️

Reply:
Can confirm...

![image](https://user-images.githubusercontent.com/379849/43509044-d909c2be-9571-11e8-8625-9661392dac35.png)

Reply:
Fint å få det likt på tvers da :+1: